### ISSUE TITLE
CNX does not properly remove remote edges.

### ISSUE THREAD

**Issue by sawyer-jack-1** [sawyer-jack-1/connection-graph-wasserstein#3]
This is a note for myself for later.

in ConnectionNetworkX.py at line 138, remote edge (i,j) is removed from the NX object. It is not witnessed further in any iterables or edge lists, counts, etc.

However (i,j) is only "zeroed out" in the connection incidence matrix. So any reference to the CNX object still contains a copy of the blocks related to the edge, but with zero weight. To fix: need to dynamically resize the cnx sparse matrices.

Bottom line: edge iterables over the nx and cnx objects do not match up as desired. This is of particular issue in any drawings: see the regularization demo notebook, last block (as of 4/17/23).
